test/cases/fixes/incompleteTypes/reactTypes/reactPropFunctionsFromCalls/original.tsx

```diff
@@ -26,11 +26,11 @@ import * as React from 'react';
     }
 
     type WithFunctionsProps = {
-        providesNothing: () => void;
-        providesString: (arg0: string) => void;
-        providesNumberThenString: (arg0: number, arg1: string) => void;
-        providesBooleanGivesNumber: (arg0: boolean) => number;
-        returnsString: () => string;
+        providesNothing: Function;
+        providesString: Function;
+        providesNumberThenString: Function;
+        providesBooleanGivesNumber: Function;
+        returnsString: Function;
```

Comment:
🤔 how unfortunate. I think the tests are hamstrung by only being able to run one round of mutation. I'll probably have to reconfigure them to run up to some arbitrary >1 number.

Reply:
(...or maybe this is just an introduced bug?)